sci-libs/silo/silo-4.11.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit fortran-2
+
+DESCRIPTION="A mesh and field I/O library and scientific database"
+HOMEPAGE="https://wci.llnl.gov/simulation/computer-codes/silo"
+SRC_URI="https://wci.llnl.gov/sites/wci/files/2021-09/${P}-bsd.tgz"
+S="${WORKDIR}/${P}-bsd"
+
+SLOT="0"
+LICENSE="BSD"
+KEYWORDS="~amd64 ~x86"
+IUSE="browser +hdf5 +silex +szip +qt5 test"
```

Comment:
From what I understand qt5 can't be disabled, so this flag should just be dropped if we need it unconditionally

Reply:
Required by Silex REQUIRED_USE

Reply:
Well it was previously, but now apparently `qt5` is always enabled so this requirement will always be satisfied